Gifski/ConversionCompletedViewController.swift

```diff
@@ -41,6 +41,9 @@ final class ConversionCompletedViewController: NSViewController {
 	override func viewDidAppear() {
 		super.viewDidAppear()
 
+		self.view.window?.makeFirstResponder(self)
```

Comment:
really interesting: so making it a first responder in `viewDidAppear` is working but the generic way in `AppDelegate` with the app became active method is not? In that case we might need to see if other VCs also need to do that in `viewDidAppear`. Really strange.

Reply:
Yes, the problem seems to be making them first responder before their view is shown. Also really strange that it works in Catalina.